packit/constants.py

```diff
@@ -177,3 +177,12 @@
     "Use it multiple times to select multiple packages."
     "Defaults to all the packages listed inside the config."
 )
+
+SYNC_RELEASE_PR_INSTRUCTIONS = (
+    "If you need to do any change in this pull request, you need to locally fetch the "
+    "source branch of it and push it (with a fix) to your fork "
+    "(as it is not possible to push to the branch created in the Packit’s fork):\n"
+    "```\ngit fetch https://src.fedoraproject.org/forks/packit/rpms/{package}.git "
+    "refs/heads/*:refs/remotes/packit/*\n"
+    "git cherry-pick packit/{branch}\n```\n"
```

Comment:
I am wondering if  `checkout` won't be easier to work with since it doesn't matter where we are when running the command.

Reply:
I don't have a strong opinion about it, I copied the same commands that are recommended in our docs: https://packit.dev/docs/configuration/downstream/pull_from_upstream/#example. Should I change both?

Reply:
Yes please. Since these commands should be for less knowledgeable people (or the lazy ones..;) I would try to make it as simple/error-prone as possible.
Thanks!

Reply:
changed, PTAL...also added the parameter for Packit user (prod vs stg)

Reply:
Good point with the user!